android/app/src/main/java/app/organicmaps/util/Utils.java

```diff
@@ -774,4 +775,15 @@ public static PackageInfo getPackageInfo(@NonNull PackageManager manager, @NonNu
       return getPackageInfoOld(manager, packageName, flags);
     return manager.getPackageInfo(packageName, PackageManager.PackageInfoFlags.of(flags));
   }
+
+  public static void openWebview(@NonNull Context context, @NonNull Intent intent){
```

Comment:
It's not a good idea to merge methods that are not used yet (and not confirmed that they are working yet). Let's split tasks and move incrementally:
1. Finish this PR with openUrl and translations
2. Make an issue for a separate task to handle missing webview crashes
3. After that, a fix can be tested/developed for webviews.